common/scala/src/main/scala/org/apache/openwhisk/core/entity/ConcurrencyLimit.scala

```diff
@@ -37,21 +40,67 @@ case class ConcurrencyLimitConfig(min: Int, max: Int, std: Int)
  * The constructor is private so that argument requirements are checked and normalized
  * before creating a new instance.
  *
- * @param maxConcurrent the max number of concurrent activations in a single container
+ * @param maxConcurrent the max or min number of concurrent activations in a single container
```

Comment:
Because it is `@param maxConcrrent`, why also has `min` meaning?

Reply:
@ningyougang my mistake, I'll update it.